tgui/packages/tgui/styles/interfaces/DirectorPanel.scss

```
@@ -0,0 +1,109 @@
+}
+
+.DirectorPanel__Table {
+  table-layout: fixed;
+  border: 1px solid rgba(96, 139, 178, 0.28);
+  border-collapse: separate;
+
+  .Table__cell {
+    padding: 0.4em 0.55em;
+    border-right: 1px solid rgba(96, 139, 178, 0.2);
+    vertical-align: top;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)